paymentsheet/src/main/java/com/stripe/android/paymentsheet/PaymentSheetViewModel.kt

```diff
@@ -123,8 +125,8 @@ internal class PaymentSheetViewModel @Inject internal constructor(
     private val _paymentSheetResult = MutableLiveData<PaymentSheetResult>()
     internal val paymentSheetResult: LiveData<PaymentSheetResult> = _paymentSheetResult
 
-    private val _startConfirm = MutableLiveData<Event<ConfirmStripeIntentParams>>()
-    internal val startConfirm: LiveData<Event<ConfirmStripeIntentParams>> = _startConfirm
+    private val _startConfirm = MutableStateFlow<Event<ConfirmStripeIntentParams>?>(null)
```

Comment:
Looks like another great candidate for `MutableShareFlow(replay = 1)`.

Reply:
Opted to remove the flow altogether